.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,130 @@
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+      checks: write
+    env:
+      NEW_SEMGREP_VERSION: ${{ github.event.inputs.version }}
+    steps:
+      - id: jwt
+        env:
+          EXPIRATION: 600
+          ISSUER: ${{ secrets.SEMGREP_CI_APP_ID }}
+          PRIVATE_KEY: ${{ secrets.SEMGREP_CI_APP_KEY }}
+        name: Get JWT for semgrep-ci GitHub App
+        uses: docker://public.ecr.aws/y9k7q4m1/devops/cicd:latest
```

Comment:
What is this image and why are we using it? Seems like this warrants a comment.

Reply:
I copied this from another similar script; but: I query this image with the secrets (issuer + private key) to get the jwt needed to get semgrep-ci[bot]'s github token + to push code as semgrep-ci[bot].